stdlib/public/Observation/Sources/Observation/Observable.swift

```diff
@@ -34,6 +63,11 @@ public macro Observable() =
 public macro ObservationTracked() =
   #externalMacro(module: "ObservationMacros", type: "ObservationTrackedMacro")
 
+/// Turns off observation tracking of a property.
```

Comment:
Is the phrase "Turns off" right? It more-so disables automatic synthesis of observation.